$OpenBSD: patch-src_compositor_meta-background_c,v 1.1 2013/03/29 16:46:14 jasper Exp $

commit 992a15e6401114a67450dcf10b080044caab6b31
Author: Jasper Lievisse Adriaanse <jasper@humppa.nl>
Date:   Fri Mar 29 16:59:26 2013 +0100

    Don't redefine MetaBackgroundPrivate, fixing a build issue:
    
    compositor/meta-background.c:64: error: redefinition of typedef 'MetaBackgroundPrivate'
    ./meta/meta-background.h:51: error: previous declaration of 'MetaBackgroundPrivate' was here
    
--- src/compositor/meta-background.c.orig	Fri Mar 29 16:49:23 2013
+++ src/compositor/meta-background.c	Fri Mar 29 16:49:35 2013
@@ -61,7 +61,6 @@
  *
  * This object provides a ClutterContent object to assist in sharing between actors.
  */
-typedef struct _MetaBackgroundPrivate MetaBackgroundPrivate;
 
 struct _MetaBackgroundPrivate
 {
